### DISCUSSION TITLE
Başarımlar

### DISCUSSION THREAD

**Discussion by enzifiri** [enzifiri/GitHub-Achievements#35] [Q&A] [answered]
Github başarımları sayesinde form doldurduğunuz projelerden seçilme ihtimalinizi az da olsa arttırabileceğinizi düşünüyorum. Profilde dikkat çekici duruyor. Bazı başarımları almak gayet kolay, nasıl alabileceğinizi anlatmaya çalısıcam. Öncelikle Başarımları bi inceleyelim.

**Comment by 0xpellegrino** [comment, marked as answer]
Eline sağlık hocam.

**Reply by enzifiri** [reply]
Telegramdan bana ulaş hocam

**Comment by 0xik** [comment]
elinize saglık hocam
